sentry/src/main/java/io/sentry/hints/Hints.java

```diff
@@ -1,29 +1,108 @@
 package io.sentry.hints;
 
+import io.sentry.Attachment;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
+import java.util.concurrent.CopyOnWriteArrayList;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
 public final class Hints {
 
   private final @NotNull Map<String, Object> internalStorage = new HashMap<String, Object>();
+  private final @NotNull List<Attachment> attachments = new CopyOnWriteArrayList<>();
+  private @Nullable Attachment screenshot = null;
```

Comment:
do we need some kind of synchronization around this field (looking at `attachments` - it's `CopyOnWriteArrayList`)

Reply:
Ah no, I don't think it needs to be synchronized, can replace with a different implementation.

Reply:
Updated